lib/paper_trail/reifier.rb

```diff
@@ -284,9 +282,8 @@ def reify_has_many_through(transaction_id, associations, model, options = {})
       # table, this method would return the constant `Fruit`.
       def version_reification_class(version, attrs)
         inheritance_column_name = version.item_type.constantize.inheritance_column
-        class_name = attrs[inheritance_column_name].blank? ?
```

Comment:
> Do people ever set inheritance_column to "" ?

Seems unlikely, but we should be changing behavior in a linter commit. If we are going to make that change, please make a separate PR. Thanks.